### PR TITLE
Add the rest of the x86 registers

### PR DESCRIPTION
**Pull request by endeav0r**
I used https://github.com/unicorn-engine/unicorn/blob/master/bindings/python/unicorn/x86_const.py#L244 as a reference

### CODE REVIEW

File: src/x86_const.rs [hunk 1]
```diff
@@ -244,6 +244,15 @@ pub enum RegisterX86 {
...
     R15W,
+    IDTR,
+    GDTR,
+    LDTR,
+    TR,
+    FPCW,
+    FPTAG,
+    MSR,
+    MXCSR,
+    ENDING
```
Comment (ekse) [inline src/x86_const.rs]: ENDING is an end marker, we don't need it. Please remove it .

Comment (endeav0r) [reply src/x86_const.rs]: removed in latest commit

**Comment by endeav0r** [comment]
I was careless, I pushed an additional change with this PR. Small change though, so hopefully it will be ok.

File: src/x86_const.rs [hunk 2]
```diff
@@ -270,3 +270,12 @@ pub enum InsnSysX86 {
     SYSCALL = InsnX86::SYSCALL as isize,
     SYSENTER = InsnX86::SYSENTER as isize,
 }
+
+#[repr(C)]
+#[derive(PartialEq, Debug, Clone, Copy)]
+pub struct X86Mmr {
+    pub selecter: u64,
```
Comment (ekse) [inline src/x86_const.rs]: typo: selector

**Comment by ekse** [comment]
It's fine, its a small PR and the changes are related. There's just a small typo to fix and it will be good to merge.

File: src/x86_const.rs (hunk 2 above)
Comment (endeav0r) [reply src/x86_const.rs]: typo fixed in https://github.com/ekse/unicorn-rs/pull/55/commits/d9795935e4c02c8e8f31ea05953818ab989e4b47

**Pull request merged**